code/drasil-data/lib/Data/Drasil/Concepts/Documentation.hs

```diff
@@ -75,6 +75,8 @@ typUnc      = commonIdeaWithDict "typUnc"      (cn' "typical uncertainty")
 sec         = commonIdeaWithDict "section"     (cn' "section")                                       "Sec"     [documentc]
 srs         = commonIdeaWithDict "srs"         softReqSpec                                           "SRS"     [softEng]
 notebook    = commonIdeaWithDict "notebook"    (cn' "notebook")                                      "NB"      [softEng]
+refBy       = commonIdeaWithDict "refBy"       (cn' "Referenced By")                                 "RefBy"   [documentc]
```

Comment:
One thing to change: we put our concepts all in lower-case, and then Drasil takes care of casing if/when needed.

Reply:
Thanks for pointing that out! I will make the change now.